### PR TITLE
Workshop improvements

### PR DESCRIPTION
**Pull request by samdoran**
Mostly documentation changes.

Relevant functional changes:
- add grafana host to inventory (e559e10)
- install `firewalld` so tasks turning it off won't fail (f546649)
- move `admin_password` to variable and store `admin_password_hash` in variable (427ac65)
- tag the email task for easy execution (757f896)
- convert `lameapp_version` to a string to prevent errors (41ff807)
- correct lookup plugin solution so it works with Ansible >= 2 (6572030)

### CODE REVIEW

**Comment by jsmartin** [comment]
+1 from me.. @defionscode

File: workshops/developer/lookup_plugins/lookup_plugins/template_url.py [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+
+from ansible.errors import AnsibleError, AnsibleParserError
+from ansible.plugins.lookup import LookupBase
+from ansible.module_utils.urls import *
+
+try:
+    from __main__ import display
+except ImportError:
+    from ansible.utils.display import Display
+    display = Display()
```
Comment (defionscode) [inline workshops/developer/lookup_plugins/lookup_plugins/template_url.py]: Why is display being imported if it's not being used?

**Comment by defionscode** [comment]
Just a small issue with `display` being imported and it not being used

**Comment by samdoran** [comment]
Not sure why that's there. I most likely copied it in not fully understanding it. I'll see if I can find the source and see what's up.

**Pull request merged**